### PR TITLE
Add more examples

### PR DESCRIPTION
**Pull request by miherlosev**

### CODE REVIEW

**Comment by miherlosev** [comment]
@VasilyStrelyaev fill the `README.md` page for the added examples.

File: examples/xpath-selectors/xpath-selector.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    while (item) {
+        items.push(item);
+        item = iterator.iterateNext();
+    }
+
+    return items;
+});
+
+export default function (xpath) {
+    return Selector(elementByXPath(xpath));
```
Comment (VasilyStrelyaev) [inline examples/xpath-selectors/xpath-selector.js]: @miherlosev So do we have to wrap `elementByXPath` in `Selector` once again?

Comment (miherlosev) [reply examples/xpath-selectors/xpath-selector.js]: Yes. It allows us to apply the Selector methods to the call result.
```js
XPathSelector('//input[@type="checkbox"]').nth(1); // It's not possible to call '.nth(1) for the nodes collection.
```

File: examples/use-xpath-selectors/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Use XPath Selectors
+
+**Test Code**: [index.js](index.js)
+
+This example shows how to use XPath selectors in TestCafe tests.
+
+The `xpath-selector.js` file exports a [TestCafe selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/) that takes an XPath string as a parameter. This selector executes a function that evaluates the `xpath` with `document.evaluate` and returns an array of elements.
```
Comment (miherlosev) [inline examples/use-xpath-selectors/README.md]: > evaluates the `xpath` with `document.evaluate`

It's not correct. Something like this: `This selector executes a function that searches elements by given xpath expression`.

**Comment by miherlosev** [comment]
LGTM

I cannot approve of my PR in the GitHub UI. 
ping @AndreyBelym

**Comment by VasilyStrelyaev** [comment]
@AndreyBelym It's still need to be checked by Dirk

**Comment by VasilyStrelyaev** [comment]
Armina approved it, we can merge now

**Pull request merged**